### PR TITLE
Convert raw Elixir version from brew into using asdf to install our pinned version

### PR DESCRIPTION
**Pull request by BartElliott**
Simplifies the later readme instructions to just run ./run.sh instead of listing each individual step.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,53 +112,38 @@ Enter it again:
...
-
-```sh
-npm ci --prefix=assets
-```
-
-Then, run the following to generate local SSL certificates (for HTTPS)
-
-```sh
-mix phx.gen.cert
-```
```
Comment (clone1018) [inline README.md:154]: Also, I wonder if we're losing some educational value by removing these. For example, I never `./run.sh` and almost always prefer running some of these steps directly depending on what I need to do. I think `./run.sh` is a great quick start task though!

File: README.md [hunk 2]
```diff
@@ -112,53 +112,38 @@ Enter it again:
...
 ```sh
 # Required dependencies
-brew install elixir imagemagick node
+brew install imagemagick node
 # Optional dependencies
 brew install librsvg 
 npm install -g svgo
 # Graphical Postgres, if you do not want a graphical Postgres, you are on your own!
-brew install --cask postgres
+brew install --cask postgres-unofficial
```
[the remark this answers is not in the document]
Comment (BartElliott) [reply README.md:135]: `brew install --cask postgres` didn't exist when I tried this morning, the closest thing was this cask in brew's caskroom, which looks like this:
image

File: README.md [hunk 3]
```diff
@@ -112,53 +112,38 @@ Enter it again:
...
 ### macOS Installation
 
-Installation is simple with [Homebrew](https://brew.sh).
+Installation is simple with [Homebrew](https://brew.sh). First, we need to install our pinned version of Elixir:
+
+```sh
+brew install asdf
+asdf plugin-add elixir
+echo -e "\n. $(brew --prefix asdf)/asdf.sh" >> ${ZDOTDIR:-~}/.zshrc
+asdf install elixir `sed -n -e 's/ *elixir: "~> \(.*\)",/\1/p' mix.exs`
```
[the remark this answers is not in the document]
Comment (BartElliott) [reply README.md:121]: done

File: README.md (hunk 1 above)
Comment (BartElliott) [reply README.md:154]: Maybe a comment, how's:
```As a first task, dig into the `run.sh` script to see the script steps it runs to install dependencies and start up the server. You won't need to rebuild all of these steps for every type of change, so figuring out which steps you need to run for your changes will speed up your dev feedback loop.```?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (clone1018) [reply README.md:135]: Ah yeah, that's the right thing then, thanks!

File: README.md (hunk 1 above)
Comment (clone1018) [reply README.md:154]: Sounds good to me!

**Comment by BartElliott** [comment]
@clone1018 changes complete, please double-check

**Review by clone1018 — APPROVED** [review]

**Pull request merged**